business/checkers/sidecars/egress_listener_checker.go

```diff
@@ -11,9 +11,10 @@ import (
 )
 
 type EgressHostChecker struct {
-	Sidecar        networking_v1alpha3.Sidecar
-	ServiceEntries map[string][]string
-	ServiceList    models.ServiceList
+	Sidecar          networking_v1alpha3.Sidecar
+	ServiceEntries   map[string][]string
+	ServiceList      models.ServiceList
+	RegistryServices []*kubernetes.RegistryService
```

Comment:
Hi @hhovsepy, just me to sync with you on this.
At some point, probably the ServiceList and the RegistryServices will converge.
Do you want to address this first and perhaps merge the list of services later ?
I know that the refactoring could be more deep in that case as well.

Reply:
I would leave that after converging and merge this first not to be blocked.